Project.toml

```
@@ -1,6 +1,6 @@
 name = "AbstractAlgebra"
 uuid = "c3fe647b-3220-5bb0-a1ea-a7954cac585d"
-version = "0.43.0-DEV"
+version = "0.42.99"
```

```suggestion
version = "0.43.0-DEV"
```
prepare for me to apply it later from my phone